website/docs/r/notification_configuration.html.markdown

```diff
@@ -124,6 +124,8 @@ The following arguments are supported:
   is `email`.
 * `workspace_id` - (Required) The id of the workspace that owns the notification configuration.
 
+-> **Note:** Write-Only argument `token_wo` is available to use in place of `token`. Write-Only arguments are supported in HashiCorp Terraform 1.11.0 and later. [Learn more](https://developer.hashicorp.com/terraform/language/v1.11.x/resources/ephemeral#write-only-arguments).
```

Comment:
I'm not sure if we came to a consensus on this, but I feel (mildly) that this note should either be supplemented with a full treatment for a `value_wo` attribute above, or at the very least a short reference in the description for `value`. Only because I think it might be easy for users to miss if they don't scroll down far enough. 

More importantly, I want to give a consistent treatment across the docs for all these new write-only attributes. I don't feel strongly about which way we decide but there are a few places I did it differently. Thoughts?